wrap/gtwrap/pybind_wrapper.py

```diff
@@ -292,6 +296,37 @@ def wrap_namespace(self, namespace):
             )
         return wrapped, includes
 
+    def wrap_pickle(self, cpp_class):
+        """Wrap to enable pickling on the generated Python class.
+        
+        Pickling is supported by reusing the serialization functionality already build in to many GTSAM types.
+
+        Ref: https://pybind11.readthedocs.io/en/stable/advanced/classes.html#pickling-support
+        """
+        if cpp_class not in [
+            "gtsam::Cal3Bundler",
+            "gtsam::PinholeCamera<gtsam::Cal3Bundler>", 
+            "gtsam::Pose3", 
+            "gtsam::Rot3", 
+            "gtsam::SfmTrack",
+            "gtsam::Unit3", 
+        ]:
+            # TODO: fix this check by just checking if serialization is supported.
+            return ""
+        return textwrap.dedent('''
```

Comment:
could you maybe add some inline comments explaining why the textwrap dedent module is needed here, and why the C++ code needs to be provided as a string in Python?

Reply:
This same structure has been used in this file. Do I still add the comments?